README.md

```diff
@@ -138,6 +138,206 @@ OS X users can install Cromwell with Homebrew: `brew install cromwell`.
 
 See the [migration document](MIGRATION.md) for more details.
 
+# Command Line Usage
+
+For built-in documentation of Cromwell command line usage, run the Cromwell JAR file with no arguments:
+
+```
+$ java -jar cromwell-<versionNumber>.jar
+```
+
+For example, `$ java -jar cromwell-29.jar`. You will get a usage message like the following:
+
+```
+cromwell 29
+Usage: java -jar /path/to/cromwell.jar [server|run] [options] <args>...
```

Comment:
ToL I struggled with this syntax section a little. I think a more accurate representation would be:
```
usage: java -jar cromwell.jar server | run [options] <workflow-source>
```